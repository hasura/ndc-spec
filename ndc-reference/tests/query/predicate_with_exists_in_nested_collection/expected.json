[
  {
    "rows": [
      {
        "id": 2,
        "name": "Chalmers University of Technology",
        "staff": [
          {
            "first_name": "John",
            "last_name": "Hughes",
            "specialities": [
              "Computer Science",
              "Functional Programming",
              "Software Testing"
            ]
          },
          {
            "first_name": "Koen",
            "last_name": "Claessen",
            "specialities": [
              "Computer Science",
              "Functional Programming",
              "Automated Reasoning"
            ]
          }
        ]
      }
    ]
  }
]
