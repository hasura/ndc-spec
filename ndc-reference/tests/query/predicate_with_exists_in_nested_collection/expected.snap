---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/predicate_with_exists_in_nested_collection/request.json
---
[
  {
    "rows": [
      {
        "id": 2,
        "name": "Chalmers University of Technology",
        "staff": [
          {
            "first_name": "John",
            "last_name": "Hughes",
            "specialities": [
              "Computer Science",
              "Functional Programming",
              "Software Testing"
            ],
            "born_country_id": 2
          },
          {
            "first_name": "Koen",
            "last_name": "Claessen",
            "specialities": [
              "Computer Science",
              "Functional Programming",
              "Automated Reasoning"
            ],
            "born_country_id": 2
          }
        ]
      }
    ]
  }
]
